lib/order.rb

```diff
@@ -0,0 +1,20 @@
+class Order
+  attr_reader :basket, :item, :quantity
```

Comment:
I don't see the need to expose the value of `item` and `quantity` to things outside the class with these attribute readers. These attribute readers at the moment will return `nil` if called, because there are no instance variables called `@item` or `@quantity`. You're correct not to make them into instance variables, as you've only used them as local variables.